### PR TITLE
doc: README updated to include information on installing and using Vagrant and Ansible on a  Windows machine

### PR DESCRIPTION
**Pull request by KalyanJoseph**
Updated the README to include sections for installing Vagrant on a Windows machine, and installing and running Ansible within the Vagrant virtual machine.

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [FAQ.md](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

File: ansible/README.md [hunk 1]
```diff
@@ -212,6 +212,10 @@ machine to run vagrant in [playbooks/vagrant.yml](playbooks/vagrant.yml) but
 it simply installs Vagrant from https://releases.hashicorp.com/vagrant/2.2.5/vagrant_2.2.5_x86_64.deb
 and also [virtualbox from their web site](https://www.virtualbox.org/wiki/Downloads)
 
+## Vagrant setup guide - Windows
+
+To test the ansible scripts, you'll need to install Vagrant and Virtualbox from [here](https://www.vagrantup.com/intro/getting-started/index.html)
+
 ## Executing under vagrant
```
Comment (karianna) [inline ansible/README.md:219]: Does this title also need to be edited to state it's Linux / MacOS X?

File: ansible/README.md [hunk 2]
```diff
@@ -249,6 +253,27 @@ In case one or more tasks fail or should not be run in the local environment, se
 ```bash
 ansible-playbook -b AdoptOpenJDK_Unix_Playbook/main.yml --skip-tags="install_zulu,jenkins_authorized_key,nagios_add_key,add_zeus_user_key"
 ```
+
+## Executing under vagrant (Windows)
+
+Ansible cannot be installed on a Windows machine, so you should boot a Linux VM using Vagrant and install it there instead:
+1) Copy a Linux Vagrantfile from  the `openjdk-infrastructure/ansible/vagrant` directory into the `openjdk-infrastructure/ansible` directory, and save it without an extension.
```
Comment (karianna) [inline ansible/README.md:260]: If you can show the example of this using powershell commands that would be a bonus:

e.g.

```powershell
copy ./openjdk-infrastructure/ansible/vagrant/Vagrantfile.Ubuntu14 ./openjdk-infrastructure/ansible
```

File: ansible/README.md [hunk 3]
```diff
@@ -249,6 +253,27 @@ In case one or more tasks fail or should not be run in the local environment, se
 ```bash
 ansible-playbook -b AdoptOpenJDK_Unix_Playbook/main.yml --skip-tags="install_zulu,jenkins_authorized_key,nagios_add_key,add_zeus_user_key"
 ```
+
+## Executing under vagrant (Windows)
+
+Ansible cannot be installed on a Windows machine, so you should boot a Linux VM using Vagrant and install it there instead:
```
Comment (karianna) [inline ansible/README.md:259]: Do you have a Markdown linter in your IDE?  Might be worth running it, typically Markdown likes an empty newline after a `:` and there will be other things like that.

**Review by karianna — CHANGES_REQUESTED** [review]
Looks good overall, some nitpicks and suggestions

File: ansible/README.md (hunk 1 above)
Comment (KalyanJoseph) [reply ansible/README.md:219]: yep, will do

File: ansible/README.md (hunk 3 above)
Comment (KalyanJoseph) [reply ansible/README.md:259]: Will do :)

File: ansible/README.md (hunk 2 above)
Comment (KalyanJoseph) [reply ansible/README.md:260]: I'll add that example and also an example of renaming to remove the extension

Comment (sxa) [reply ansible/README.md:260]: Make sure the slashes are the appropriate way round e.g. `\` if using the Windows `copy` command and `/` if using `cp`

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**